src/lib/registration/registration.rb

```diff
@@ -136,7 +136,11 @@ def update_system(target_distro = nil)
     # @return [Array<Addon>] List of addons, empty if no base product is found
     def get_addon_list
       # extensions for base product
-      base_product = ::Registration::SwMgmt.base_product_to_register
+      base_product = if Yast::Mode.update
```

Comment:
Um, we need to be careful that the migration can run also in an installed system. An in that case the mode is very likely *not* set to update. We should check that.

Reply:
well, at least for now it is not regression as originally only one call was there